### DISCUSSION TITLE
Systemd messages

### DISCUSSION THREAD

**Discussion by arcolinuxz** [picodotdev/alis#226] [General]
Is this something we should be concerned about?
Is this something we should change in the scripts of Alis?

![systemd-boot message](https://user-images.githubusercontent.com/36679406/196865368-2d2a9975-133f-4105-ad80-7fb9b3ffad6c.png)

**Comment by picodotdev** [comment]
I did not get that message on my last systemd-boot update, anyway I reviewed the systemd-boot wiki and changed the command to update systemd-boot as the wiki suggest to do this.

https://github.com/picodotdev/alis/commit/24fe21010a2700876dac57a9ea795cfb0fc2fb71

**Comment by arcolinuxz** [comment]
thanks
